### ISSUE TITLE
Trading fee schedule

### ISSUE THREAD

**Issue by ElectricDosh** [The-GNTL-Project/Exchange#76]
Trading fees are a crucial part of making a profitable exchange, so must be thought of with the long-term in mind. Large, unwavering, or unclear fees are liable to scare off newcomers in the short-term and limit profitability in the long-term. 

At the same time, a well-thought out fee system can be both lucrative and make users feel like they're getting a bargain, encourage brand loyalty, and ultimately be the driving force behind financing more and more growth for the exchange.

I think a fairly long promotional period from the launch onward with reduced maker fees for everyone would be a great way to kickstart the order books, with an eye to implementing a tiered fee system based on trade volume in the long-term. 

If you have other suggestions please share them here.

**Comment by bennytehcat** [comment]
Good idea, similar spirit to what I suggested in #55. Anything new should be "cheaper" or free.  After initial launch, we should make a fee schedule similar to other platforms.  It is usually based on satisfying one of two requirements:
 
A) Holding "x" amount of "y" coin for lower fees  (Good use for GNTL Coin, perhaps.)
B) Executing $x worth of trades on a monthly basis

**Comment by ElectricDosh** [comment]
Definitely a good idea to encourage stacking GNTL! Where I think we should differ from the other exchanges is on the minimum amounts required for the first VIP level. For example on Binance the minimum required to make VIP 1 is 50 BTC trade volume in a month, which I doubt most people would hit in a lifetime, let alone a month. Their entire VIP system is just noise to the average user,

If 0.2% is the desired "standard" fee, I'd suggest something like having 0.25% be the base fee at VIP level 0, which drops to 0.2% at VIP 1 at something trivial like $100 a month traded or $100 worth of GNTL staked. This gives just about everyone the chance to get that first level and find the want to maintain it every month. VIP 2 can be as high as you like after that, I think just having the first step be attainable so everyone can engage with the VIP system in some way would be enough.
